src/core/src/GltfUtil.cpp

```diff
@@ -317,21 +317,19 @@ const CesiumGltf::ImageCesium& getImageCesium(const CesiumGltf::Model& model, co
 }
 
 std::pair<std::string, uint64_t> parseAttributeName(const std::string& attributeName) {
-    const auto regex = std::regex("^([a-zA-Z_]*)(_([1-9]\\d*|0))?$");
-
-    std::string semantic;
-    std::string setIndex;
-
-    std::smatch matches;
+    std::string semantic = "";
```

Comment:
I know it's not intuitive but you should just use `{}` instead. Much less expensive for an empty string. https://godbolt.org/z/nKx8c7EP9

```suggestion
    std::string semantic{};
```

Reply:
Ah yes, my recent multi-year stint in C# sometimes leaves me forgetting I can use direct initialization. Good call.